tools/perf-automation/Azure.Sdk.Tools.PerfAutomation/Azure.Sdk.Tools.PerfAutomation.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Version>0.1.0</Version>
+
+    <TargetFramework>net6.0</TargetFramework>
+
+    <OutputType>Exe</OutputType>
+    <PackAsTool>true</PackAsTool>
+    <ToolCommandName>perf-automation</ToolCommandName>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="CommandLineParser" Version="2.8.0" />
```

Comment:
Depending on how complex the flags get, dragonfruit is a good cli parser option as well.

Reply:
I don't remember the exact scenarios, but I've seen some cases where `CommandLineParser` supported flags that would be difficult or impossible with `System.CommandLine[.DragonFruit]`.  If we wanted to standardize on one or the other for EngSys tools, I can try to remember/rediscover these scenarios.